src/modules/sensors/vehicle_air_data/VehicleAirData.cpp

```diff
@@ -469,4 +472,74 @@ void VehicleAirData::PrintStatus()
 	}
 }
 
+bool VehicleAirData::BaroGNSSAltitudeOffset()
+{
+	param_t param_hgt_ref = param_find("EKF2_HGT_REF");
```

Comment:
I see your point, but I don't think this module should look for a param from the EKF2 module.
We have [SENS_MAG_AUTOCAL](https://github.com/PX4/PX4-Autopilot/blob/c2f872c558932a6d3c8b03497a28b05da92c1046/src/modules/sensors/sensor_params_mag.c#L119-L129) and [SENS_IMU_AUTOCAL](https://github.com/PX4/PX4-Autopilot/blob/c2f872c558932a6d3c8b03497a28b05da92c1046/src/modules/sensors/vehicle_imu/imu_parameters.c#L52-L62), why not `SENS_BARO_AUTOCAL` ?